src/internal/m365/collection/drive/collections_tree.go

```diff
@@ -810,13 +831,13 @@ func (c *Collections) turnTreeIntoCollections(
 			uc,
 			c.counter.Local())
 		if err != nil {
-			return nil, nil, clues.StackWC(ctx, err)
+			return nil, nil, nil, clues.StackWC(ctx, err)
 		}
 
 		coll.driveItems = cbl.files
 
 		collections = append(collections, coll)
 	}
 
-	return collections, newPrevPaths, el.Failure()
+	return collections, newPrevPaths, tree.generateExcludeItemIDs(), el.Failure()
```

Comment:
this is sixes, but it seems like this could be moved to the function that calls this instead of here if we wanted fewer return parameters